Source/ocdm/adapter/gstreamer/open_cdm_adapter.cpp

```diff
@@ -2,7 +2,7 @@
  * If not stated otherwise in this file or this component's LICENSE file the
  * following copyright and licenses apply:
  *
- * Copyright 2020 RDK Management
+ * Copyright 2020 Metrological
```

Comment:
Pls restore Copyright to RDK Management.

Reply:
Done